test/unit/storage/keyvalue/WrappedExpiringStorage.test.ts

```diff
@@ -141,33 +145,10 @@ describe('A WrappedExpiringStorage', (): void => {
     // Await the function that should have been executed by the interval
     await (mockInterval.mock.calls[0][0] as () => Promise<void>)();
 
+    // Make sure timer.unref() is called on initialization
+    expect(mockFn).toHaveBeenCalledTimes(1);
```

Comment:
This is not what is being tested here. Here you test that `setInterval` is called on initialization (which is also good to test though). To test that `timer.unref` is called as well, you would have to test the function that is assigned to `unref` above. 

So to make this comment be correct, you would first need to define the mock object above separately, e.g.
```ts
const mockTimer = { unref: jest.fn() };
const mockFn = jest.fn().mockReturnValueOnce(mockTimer);
```

and then add a test here to check that
```ts
expect(mockTimer.unref).toHaveBeenCalledTimes(1);
```